core/src/main/scala/cats/syntax/transLift.scala

```diff
@@ -6,5 +6,24 @@ trait TransLiftSyntax {
 }
 
 final class TransLiftOps[M[_], A](val ma: M[A]) extends AnyVal {
-  def liftT[MT[_[_],_]](implicit TL: TransLift[MT, M]): MT[M,A] = TL.liftT(ma)
+  def liftT[MT[_[_],_]](implicit extract: TLExtract[MT, M]): MT[M, A] = extract.TL.liftT(ma)(extract.TC)
 }
+
+trait TLExtract[MT[_[_], _], M[_]] {
+  val TL: TransLift[MT]
+  val TC: TL.TC[M]
```

Comment:
Should these be `def`s instead of `val`s? I get uneasy about abstract `val`s in `trait`s.